tests/indexes/tests.py

```diff
@@ -223,6 +223,34 @@ def test_ops_class_descending_partial(self):
             cursor.execute(self.get_opclass_query % indexname)
             self.assertCountEqual(cursor.fetchall(), [('text_pattern_ops', indexname)])
 
+    def test_create_index_with_desc_suffix(self):
+        # Define the index
+        desc_index = Index(
+            fields=['-body'],
+            name='whitespace_idx',
+            opclasses=['text_pattern_ops'],
+        )
+
+        # Create the index
+        with connection.schema_editor() as editor:
+            sql = desc_index.create_sql(IndexedArticle2, editor)
```

Comment:
I think we can use `Article.headline`, chop blank lines, and simplify these tests, e.g.
```python
def test_create_index_with_desc_suffix(self):
    index = Index(
        fields=['-headline'],
        name='whitespace_idx',
        opclasses=['text_pattern_ops'],
    )
    with connection.schema_editor() as editor:
        self.assertIn(
            '(%s text_pattern_ops DESC)' % editor.quote_name('headline'),
            str(index.create_sql(Article, editor)),
        )
```

Reply:
Much better, thanks!